buildSrc/build.gradle

```diff
@@ -16,31 +16,29 @@ ext {
 
 dependencies {
     //gRPC
-    compile 'io.netty:netty-tcnative-boringssl-static:2.0.54.Final'
-    compile "io.grpc:grpc-netty:$grpcVersion"
-    compile "io.grpc:grpc-protobuf:$grpcVersion"
-    compile "io.grpc:grpc-stub:$grpcVersion"
+    implementation 'io.netty:netty-tcnative-boringssl-static:2.0.54.Final'
```

Comment:
Changing from compile -> implementation is necessary but the dependency management isn't handled exactly the same. I guess I need to update the documentation about required dependencies.

Do you have a list of dependencies you needed to define in your projects build.gradle in order to get all dependencies to resolve?

Reply:
I did not have to change anything in my test project.

In my test project now most of the dependencies are (transitively) added to the `runtimeClasspath`, but not to `compileClasspath` anymore. Which is fine for my test project.